Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -1582,7 +1582,7 @@ void WindowManager::process_key_event(KeyEvent& event)
         return;
     }
 
-    if (event.type() == Event::KeyDown && event.key() == Key_Super) {
+    if (event.type() == Event::KeyDown && event.key() == Key_Super && !m_current_window_stack->active_window()->should_ignore_super_key()) {
```

Comment:
Further down there's a check for 

`if (event.type() == Event::KeyDown && event.modifiers() == Mod_Super) {`

for when you do e.g. `Super+H` or `Super+Left`. 

I fell like perhaps this fix is not "final" enough? This check is if you pressed only Super.

Reply:
That would make the shortcuts not work, right?. Which, in the context of ignore super key, I see that not being final. But also I was thinking more, don't mess with the shortcuts, just ignore super key. Maybe that should be made more clear?

For example, because this was mostly made to fix KeyboardMapper,  Super+H and Super+Left still work in KeyboardMapper without causing any issues unlike plain Super.